### PR TITLE
Add acknowledgment guide lines

### PR DESCRIPTION
**Pull request by dsoliveir**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,3 +94,11 @@ Finally, run `jupyter lab` to open a Jupyter server in the TCV-X21 repository. T
 
 To ensure that the results are reproducible, the `environment.yml` file has pinned dependencies. However, if you want to use
 this software as a library, pinned dependencies are unnecessarily restrictive. You can remove the versions after the `=` sign in the `environment.yml`, but be warned that things might break.
+
+### Acknowledgement guidelines
```
Comment (TBody) [inline README.md]: Merge with `License and attribution notice`

File: README.md [hunk 2]
```diff
@@ -94,3 +94,11 @@ Finally, run `jupyter lab` to open a Jupyter server in the TCV-X21 repository. T
...
+
+If you use any of the content of the TCV-X21 repository in one of your publications and/or talks, please cite (DOI`s link?) and properly place the acknowledgements as instructed in bellow. We would also appreciate it if you informed us about your work by sending us an email to spc.data@epfl.ch. This will let us keep track of how the dataset is being used.
```
Comment (TBody) [inline README.md]: Correct DOI is [10.1088/1741-4326/ac4cde](doi.org/10.1088/1741-4326/ac4cde)

You can add links in Markdown with `[link text](url)`

Comment (TBody) [inline README.md]: "in bellow" to "below"

File: README.md [hunk 3]
```diff
@@ -94,3 +94,11 @@ Finally, run `jupyter lab` to open a Jupyter server in the TCV-X21 repository. T
...
 this software as a library, pinned dependencies are unnecessarily restrictive. You can remove the versions after the `=` sign in the `environment.yml`, but be warned that things might break.
+
+### Acknowledgement guidelines
+
+If you use any of the content of the TCV-X21 repository in one of your publications and/or talks, please cite (DOI`s link?) and properly place the acknowledgements as instructed in bellow. We would also appreciate it if you informed us about your work by sending us an email to spc.data@epfl.ch. This will let us keep track of how the dataset is being used. 
+
+If you use the TCV-X21 experimental and/or simulation dataset in your article, please use the following acknowledgement: "This work made use of the version X of the TCV-X21 experimental and/or simulation dataset(s) provided by the teams(s) Y and publicly released under the CC-BY 4.0 license.". The teams to be used in Y responsible for each of the datasets are highlighted in boldface in the file `tcvx21/template_acknowledgement.pptx`. The version of the dataset to be used in X is in ...?
```
Comment (TBody) [inline README.md]: `tcvx21` is the folder containing the Python library. We should store the acknowledgements in a separate folder, i.e. `acknowledgements`. Also, you can add relative links to links in the repository using `[link text](relative path)`

File: README.md [hunk 4]
```diff
@@ -94,3 +94,11 @@ Finally, run `jupyter lab` to open a Jupyter server in the TCV-X21 repository. T
...
+
+If you use the TCV-X21 experimental and/or simulation dataset in a talk, please use the layout for acknowlegdements in `tcvx21/template_acknowledgement.pptx` including the only the teams that were involved in the preparation of the dataset that you use.
```
Comment (TBody) [inline README.md]: Need to add file, suggest in `acknowledgements` folder

File: README.md (hunk 3 above)
Comment (TBody) [reply README.md]: The version will correspond to both the Zenodo version and the Github release version. Note that there are separate DOIs for the Zenodo repository "all versions" [10.5281/zenodo.5776285](https://doi.org/10.5281/zenodo.5776285) and for a specific version i.e. 1.0.0 [10.5281/zenodo.5776286](https://doi.org/10.5281/zenodo.5776286)

**Comment by TBody** [comment]
Please add a description of the changes made, and enable actions in your fork

**Comment by dsoliveir** [comment]
All good

**Pull request closed without merging**